packages/components/form-field/cleaner.ts

```diff
@@ -11,13 +22,29 @@ import { KbqIconModule } from '@koobiq/components/icon';
     selector: 'kbq-cleaner',
     exportAs: 'kbqCleaner',
     template: `
-        <i [autoColor]="true" color="contrast-fade" kbq-icon-button="kbq-xmark-circle_16"></i>
+        <ng-content>
+            <i [autoColor]="true" [color]="'contrast-fade'" kbq-icon="kbq-xmark-circle_16"></i>
+        </ng-content>
     `,
     styleUrls: ['cleaner.scss'],
     host: {
-        class: 'kbq-cleaner'
+        class: 'kbq-cleaner',
+        '[attr.tabindex]': 'tabindex'
     },
     encapsulation: ViewEncapsulation.None,
     changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class KbqCleaner {}
+export class KbqCleaner implements AfterViewInit, OnDestroy {
+    protected readonly nativeElement = inject(ElementRef).nativeElement;
+    protected readonly focusMonitor = inject(FocusMonitor);
+
+    @Input({ transform: numberAttribute }) tabindex: number = 0;
```

Comment:
в большинстве мест index пишем с большой буквы
```suggestion
    @Input({ transform: numberAttribute }) tabIndex: number = 0;
```

Reply:
в идеале, в будущем нужно вынести это в хост директиву, чтобы не было ломающих изменений, лучше придерживаться единообразия в нейминге

Reply:
нужно с маленькой, что бы не было путаницы https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Global_attributes/tabindex

`[attr.tabindex]': 'tabIndex'`